packages/@okta/vuepress-theme-prose/components/SidebarItem.vue

```diff
@@ -13,40 +11,37 @@
             @click="navigate"
             :class="route.path === $route.path ? 'router-link-active' : ''"
             :aria-current="route.path === $route.path && 'page'"
-            ><slot>{{ link.title }}</slot></a
-          >
-        </router-link>
-      </div>
+            >
+            <slot>
+              <span class="text-holder">
+                {{ link.title }}
+              </span>
+            </slot>
+          </a>
+    </router-link>
 
-      <div v-if="entityType === types.blankDivider">
+    <div v-if="entityType === types.blankDivider">
         <div class="blank-divider">
           {{link.title}}
         </div>
-      </div>
+    </div>
 
-      <div v-if="entityType === types.parent">
-        <div
+    <div
+          v-if="entityType === types.parent"
           :class="{
-            'is-link': true,
-            'item-collapsable': true,
+            'tree-nav-link': true,
             'children-active': link.iHaveChildrenActive
           }"
           @click="toggleExpanded"
         >
-          <svg viewBox="0 0 320 512" v-if="link.subLinks && !sublinksExpanded">
-            <path
-              d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
-            />
-          </svg>
-
-          <svg viewBox="0 0 320 512" v-if="link.subLinks && sublinksExpanded">
-            <path
-              d="M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z"
-            />
-          </svg>
-          {{ link.title }}
-        </div>
-      </div>
+          <i :class="{
+            'fa': true,
+            'fa-chevron-right': link.subLinks && !sublinksExpanded,
+            'fa-chevron-down': link.subLinks && sublinksExpanded,
```

Comment:
What does `fa` mean? Can we use something more descriptive?

Reply:
FA is class for [font-awesome icons](https://fontawesome.com/). We do not have any icon pack, so I've used that as free option, when redesign of home page was happening